### PR TITLE
Refactor api fetch

### PR DESCRIPTION
**Pull request by Patfindley**
## Is this a fix or a feature?

-fix

## What is the change?

-refactor API fetch file

## What does it fix?

-Add variable for API destination path
-Condense two similar API fetches into one function!

## Where should the reviewer start?

-APIFetch.js 
.then(App.js)

see what I did there^^

## How should this be tested?

-click a movie for its info!

### CODE REVIEW

File: src/APIFetch.js [hunk 1]
```diff
@@ -1,27 +1,7 @@
...
-  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/videos`)
-  .then(response => {
-    if (response.ok) {
-      return response.json()
-    } else {throw new Error('Oh no! We couldnt find the trailer you were looking for!')
-
-    }
-  })
-}
+const movieApi = "https://rancid-tomatillos.herokuapp.com/api/v2"
```
Comment (bhohnco) [inline src/APIFetch.js:1]: Good call making this a variable

File: src/components/App/App.js [hunk 2]
```diff
@@ -30,12 +30,11 @@ class App extends Component {
...
-    getSingleMovie(id)
-    .then(data => this.setState({selectedMovie: data.movie}))
-    .catch(error => this.setState({error: error}))
-
-    getSingleMovieTrailer(id)
-    .then(data => this.setState({selectedMovieTrailer: data.videos}))
+  getSelectedMovie(id)
+    .then(data => this.setState({
+      selectedMovie: data.selectedMovieDetails,
+      selectedMovieTrailer: data.selectedMovieTrailer
```
Comment (bhohnco) [inline src/components/App/App.js:36]: Looks clean

**Review by bhohnco — APPROVED** [review]

File: src/APIFetch.js (hunk 1 above)
Comment (sertmer) [reply src/APIFetch.js:1]: agreed! Cleans it all up a little bit

File: src/APIFetch.js [hunk 3]
```diff
@@ -30,3 +10,31 @@ export const getMovies = () => {
...
+          return response.json()
+        } else {
+          throw new Error(`Uh oh! We can't find the movie you're looking for!`)
+        }
+      })
+
+  const selectedMovieTrailer = fetch(`${movieApi}/movies/${id}/videos`)
+    .then(response => {
+      if (response.ok) {
+        return response.json()
```
Comment (sertmer) [inline src/APIFetch.js:27]: The first block of each fetch look similar to one another - have you considered just sending the whole URL through as an argument - might let you condense the many fetches down into one

**Review by sertmer — COMMENTED** [review]
Looking good, yall! I left a couple of comments with some questions, but feel free to take them or leave them!

**Pull request merged**